doc/source/programs/gdalwarp.rst

```diff
@@ -394,6 +394,9 @@ with control information.
     the "warp memory" allowed it will subdivide the chunk into smaller chunks
     and try again.
 
+    A -wm value of "unlimited" will prevent this chunking. If sufficient
```

Comment:
we should though add a word of caution that "bigger is not necessarily better" and that people should only use that if they know what they are doing / that the default is very reasonable for most use cases (well maybe you have a different experience with your tests?). Even if people have enough RAM, there are definitely situations where a too big warping memory has undesirable consequences of processing speed, particularly for scenarios that might involve reprojection between polar projections and geographic where a very small target source window can translate to a huge source window